migrate.py

```diff
@@ -373,8 +380,8 @@ def push_github_issue(issue, comments, github_repo, auth, headers):
     https://github.com/nicoddemus/bitbucket_issue_migration/issues/1
     """
     issue_data = {'issue': issue, 'comments': comments}
-    url = 'https://api.github.com/repos/{repo}/import/issues'.format(
-            repo=github_repo)
+    url = 'https://api.github.com/repos/{owner}/{repo}/import/issues'.format(
```

Comment:
See comment above. This is the second PR it's come up on, so looks like an inline comment should be added both places.